test/modules/carto/carto-bqtiler-layer.spec.js

```diff
@@ -15,7 +15,7 @@ test('CartoBQTilerLayer', t => {
 });
 
 test('CartoBQTilerLayer#updateTileJSON', t => {
-  const fetch = mockFetchWithTileJSON();
+  const fetch = mockFetchMapsV2();
```

Comment:
Shall we rename this const for something different than `fetch`? It sounds like the `window.fetch` method.

Reply:
Why? We're mocking global fetch method

Reply:
Just suggesting not calling const names like `window` methods. It's okey anyway!

Reply:
Ok, makes sense